src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="form__wrapper">
```

Comment:
you really don't need this wrappers. you can just set vertical margins for the first form:
```css
margin: 20px 0;
```